azure-quantum/tests/integration/test_streaming_problem.py

```diff
@@ -0,0 +1,109 @@
+
+#!/bin/env python
+# -*- coding: utf-8 -*-
+##
+# test_problem.py: Checks correctness of azure.quantum.optimization module.
+##
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+##
+
+## IMPORTS ##
+
+import unittest
+import json
+import os
+from typing import List
+
+from azure.quantum import Workspace
+from azure.quantum.optimization import StreamingProblem, Problem, ProblemType, Term
+from azure.quantum.storage import download_blob
+from azure.common.credentials import ServicePrincipalCredentials
+from azure.identity import DefaultAzureCredential
+
+def _init_ws_():
+    workspace = Workspace(
+        subscription_id=os.environ.get('TEST_WORKSPACE_SUB'), 
```

Comment:
The following environment variables should be used since they are already standardized, instead of defining new env vars like `TEST_WORKSPACE_SUB`.

    client_id = os.environ.get("AZURE_CLIENT_ID","")
    client_secret = os.environ.get("AZURE_CLIENT_SECRET","")
    tenant_id = os.environ.get("AZURE_TENANT_ID","")
    resource_group = os.environ.get("RESOURCE_GROUP","")
    subscription_id = os.environ.get("SUBSCRIPTION_ID","")
    workspace_name = os.environ.get("WORKSPACE_NAME","")

Reply:
Just pushed this change - @vxfield please let me know if this looks good now